packages/components/src/utils/logger.ts

```diff
@@ -1,18 +1,18 @@
 // @ts-strict-ignore
 import { LuminaJsx } from "@arcgis/lumina";
+import { LitElement } from "lit";
```

Comment:
You should import `LitElement` from `@arcgis/lumina`. This should help avoid the `{ el: Element }` intersection.